meta-lxatac-bsp/recipes-bsp/barebox/files/patches/0102-video-Add-of_get_display_timing.patch

```diff
@@ -1,61 +0,0 @@
-From: Philipp Zabel <p.zabel@pengutronix.de>
-Date: Mon, 27 Mar 2023 14:17:09 +0200
-Subject: [PATCH] video: Add of_get_display_timing
-
-Add a port of the kernel of_get_display_timing() that writes a
-struct fb_videomode instead of struct display_timing, which we
-don't have.
-
-Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
-Reviewed-by: Ahmad Fatoum <a.fatoum@pengutronix.de>
----
- drivers/video/of_display_timing.c | 22 ++++++++++++++++++++++
- include/fb.h                      |  2 ++
- 2 files changed, 24 insertions(+)
-
```

Comment:
I'd mention this change as:
patch 0102-video-Add-of_get_display_timing was included in v2023.05.0 as commit 9190f984ec2c

Similar for the other dropped patches.

Reply:
Would you be okay with me not digging through old patches to add these annotations now?